jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/actions/UpdateFunctionAction.kt

```diff
@@ -19,14 +21,20 @@ abstract class UpdateFunctionAction(private val mode: EditFunctionMode, title: S
         val project = e.getRequiredData(PlatformDataKeys.PROJECT)
 
         // Fetch latest version just in case
```

Comment:
Move this comment after the `executeOnPooledThread`

Reply:
Fixed